docs/developer-guides/contributing.md

```diff
@@ -38,3 +40,52 @@ If you would like to contribute to the project, please follow below steps of joi
 3. When you finish the feature, please fetch the latest code from origin repo, merge to your branch and resolve conflict.
 4. Submit pull requests to origin main branch.
 5. Please note that there might be comments or questions from reviewers. It will need your help to update the pull request.
+
+
+### Contribute Benchmark Results
+
+If you want to contribute benchmark results run by specified SuperBench version, please follow below guidelines.
+
+#### Where to submit
+
+All the results are stored under `results` folder. The directory structure is as follows. Please create `<your-benchmark-folder>` to submit results.
+
+.
+└── results
+    ├── v0.2
+    │   └── your-benchmark-foldername
+    │       ├── LICENSE.md
+    │       ├── README.md
+    │       ├── configs
+    │       │   ├── config1.yaml
+    │       │   └── config2.yaml
+    │       ├── results
+    │       │   ├── result1.json
+    │       │   └── result2.json
+    │       └── systems
+    │           ├── system1.json
+    │           └── system2.json
+    └── v0.3
+        └── your-benchmark-foldername
+            ├── LICENSE.md
+            ├── README.md
+            ├── configs
+            │   ├── config1.yaml
+            │   └── config2.yaml
+            ├── results
+            │   ├── result1.json
+            │   └── result2.json
+            └── systems
+                ├── system1.json
+                └── system2.json
+
+#### Files to provide
+
+Besides `README` and `LICENSE` file, you should provide at least three benchmarking related files.
+* `system.json`: This file lists all the system configuration list in json format.
+* `config.yaml`: This file is the config file to run benchmarks.
+* `result.json`: This file contains the results run by SuperBench with system configuation listed in `system.json` file.
```

Comment:
Do we need to give details for these files?

Reply:
I think we can add the detail example later when we generate the initial results.